backend/experiment/rules/base.py

```diff
@@ -152,29 +152,29 @@ def get_questionnaire(self, session, randomize=False, cutoff_index=None):
             ))
         return trials
 
-    def social_media_info(self, experiment, score):
-        current_url = f"{settings.RELOAD_PARTICIPANT_TARGET}/{experiment.slug}"
+    def social_media_info(self, block, score):
```

Comment:
I think eventually the `social_media_info` will be associated with the `Experiment` object. Just saying that this will need another refactor later on.

Reply:
I could add a comment that deprecates this function. But I could also create a [deprecation decorator](https://stackoverflow.com/a/48632082/4496102) that we can use in the future for deprecating functions which will then also warn us of not using deprecated functions and point is in to the correct way of doing things. What do you think?